.github/workflows/tests-conda.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-2019]
-        python: ["3.9", "3.10"]
+        python: ["3.9", "3.10", "3.11"]
```

Comment:
We could also remove 3.9 in general and only use that with the "minimal" environment, to keep the number of builds the same.